src/hotspot/share/gc/shared/gcConfiguration.cpp

```diff
@@ -22,9 +22,9 @@
  *
  */
 #include "gc/shared/collectedHeap.hpp"
-#include "gc/shared/gc_globals.hpp"
 #include "gc/shared/gcArguments.hpp"
 #include "gc/shared/gcConfiguration.hpp"
+#include "gc/shared/gc_globals.hpp"
```

Comment:
I think the manual sorting order placed _ before letter. Your tool undos that. I think this is OK, but I wanted to point this out so that this is done intentionally.